<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewNeuronForm</class>
 <widget class="QWidget" name="NewNeuronForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Neuron</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Neuron Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Create</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>C&amp;ancel</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Universal measures</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>201</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_IP">
      <property name="text">
       <string>IP</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="lineEdit_Vrh"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_Vrh">
      <property name="text">
       <string>Vrh</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLineEdit" name="lineEdit_a"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_tauinh">
      <property name="text">
       <string>tau inh</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_tauInh"/>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="lineEdit_At"/>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_b">
      <property name="text">
       <string>b</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_Iexc"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_V"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_tauexc">
      <property name="text">
       <string>tau exc</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_R">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_tauExc"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Amount">
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_PosY">
      <property name="text">
       <string>Pos Y</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_a">
      <property name="text">
       <string>a</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEdit_Vth"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Label"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_Vr">
      <property name="text">
       <string>Vr</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="lineEdit_R"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Amount"/>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_At">
      <property name="text">
       <string>At</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_Iexec">
      <property name="text">
       <string>I exc</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Label</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_PosX">
      <property name="text">
       <string>Pos X</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_Vr"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_PosX"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_tauV"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_V">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_PosY"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_tauV">
      <property name="text">
       <string>tau V</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLineEdit" name="lineEdit_b"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_Iinh"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_Vth">
      <property name="text">
       <string>Vth</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Ip"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_Iinh">
      <property name="text">
       <string>I inh</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="label_d">
      <property name="text">
       <string>d</string>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QLineEdit" name="lineEdit_d"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Select Model</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Label</tabstop>
  <tabstop>lineEdit_Ip</tabstop>
  <tabstop>lineEdit_Amount</tabstop>
  <tabstop>lineEdit_PosX</tabstop>
  <tabstop>lineEdit_PosY</tabstop>
  <tabstop>lineEdit_V</tabstop>
  <tabstop>lineEdit_Iexc</tabstop>
  <tabstop>lineEdit_Iinh</tabstop>
  <tabstop>lineEdit_tauExc</tabstop>
  <tabstop>lineEdit_tauInh</tabstop>
  <tabstop>lineEdit_tauV</tabstop>
  <tabstop>lineEdit_R</tabstop>
  <tabstop>lineEdit_Vr</tabstop>
  <tabstop>lineEdit_Vrh</tabstop>
  <tabstop>lineEdit_Vth</tabstop>
  <tabstop>lineEdit_At</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>graphicsView_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
